src/lib/holocene/button.svelte

```diff
@@ -44,7 +44,7 @@
     <slot />
     {#if count > 0}
       <Badge
-        class="badge absolute top-0 right-0 origin-bottom-left origin-bottom-left translate-y-[-10px] translate-x-[10px]"
+        class="badge absolute top-0 right-0 origin-bottom-left translate-y-[-10px] translate-x-[10px]"
```

Comment:
I don't love that we're hardcoding values here, but you didn't do that. My. bigger question is do we know _why_ we were using `origin-bottom-left` and are we sure it's not going to break something else?

Reply:
`origin-bottom-left` was a duplicate class in this case, I think the formatter caught it. It's the positioning of the count badge if you pass a count prop to a button.